server/data/localMockData/learnerGoalsMock.ts

```diff
@@ -1,20 +1,19 @@
 import { LearnerGoals } from '../../interfaces/learnerGoals'
 
-export const LearnerGoalsMock: LearnerGoals = {
-  prn: 'G6123VU',
-  employmentGoals: ['string'],
```

Comment:
Here is where we previously had arrays of the word `string`. The new version uses arrays of (reasonably) sensible names that better reflect what the property is containing.